L/libblastrampoline/libblastrampoline@5.15/build_tarballs.jl

```diff
@@ -0,0 +1,14 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+version = v"5.15.0"
+
+include("../common.jl")
+
+sources = lbt_sources(version)
+
+# Build the tarballs, and possibly a `build.jl` as well.
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies;
+               julia_compat="1.13",  preferred_llvm_version=llvm_version
```

Comment:
```suggestion
               julia_compat="1.12",  preferred_llvm_version=llvm_version
```
This would allow us upgrade lbt in v1.12 without much hassle in case we need to